extensions/sequelize/src/sequelize/sequelize.repository.base.ts

```diff
@@ -113,6 +115,29 @@ export class SequelizeCrudRepository<
     'sqlite3',
   ] as const;
 
+  /**
+   * Length of the `nanoid` generated for defaultFn's `shortid` and `nanoid` aliases.
+   */
+  public NANO_ID_LENGTH = 9;
+
+  /**
+   * The alias registry for `defaultFn` option used in model property definition.
+   *
+   * See: https://loopback.io/doc/en/lb4/Model.html#property-decorator
+   */
+  protected defaultFnRegistry: Record<string, () => string | number> = {
+    guid: () => uuid.v1(),
+    uuid: () => uuid.v1(),
+    uuidv4: () => uuid.v4(),
+    now: () => new Date().toJSON(),
```

Comment:
Would it be better for "now" to return a Date object instead of an ISO string? I'm not sure if it's relevant, but I observed that both the current Loopback Model and Sequelize are utilizing a Date object for "now": https://github.com/loopbackio/loopback-datasource-juggler/blob/7184fb054abe50842ac5b02a175f52397234aa97/lib/model.js#L322, https://github.com/sequelize/sequelize/blob/e91f509ee4137ed78c867a2d9a1a8d022f0087e4/packages/core/src/utils/dialect.ts#L27.

Reply:
I've changed this piece of code to use the `DataTypes.*` provided by sequelize. And the behaviour matches with LoopBack.

Thanks for the comment.